src/Components/AnimationEditorPage.js

```diff
@@ -38,19 +100,328 @@ export const AnimationEditorPage = props => {
+                setModalTitleVisible(true);
+              }}
+            >
+              <Text style={styles.textStyle}>{`${I18n.t('animationEditor.drillManager.rename')}`}</Text>
+            </TouchableHighlight>
+
+            {/* Send to team button */}
+            <TouchableHighlight
+              style={{ ...styles.openButton, backgroundColor: '#2196F3' }}
+              onPress={() => {
+                setModalManagerVisible(false);
+                doShare();
+              }}
+            >
+              <Text style={styles.textStyle}>{`${I18n.t('animationEditor.drillManager.cta')}`}</Text>
+            </TouchableHighlight>
+
+            {/* New animation button */}
+            <TouchableHighlight
+              style={{ ...styles.openButton, backgroundColor: '#2196F3' }}
+              onPress={() => {
+                createNewAnimation();
+              }}
+            >
+              <Text style={styles.textStyle}>{`${I18n.t('animationEditor.drillManager.newAnimation')}`}</Text>
+            </TouchableHighlight>
+
+            {/* Load button */}
+            {props.customeAnimations !== undefined &&
+            props.customeAnimations !== null &&
+            props.customeAnimations.length > 0 ? (
+              <TouchableHighlight
+                style={{ ...styles.openButton, backgroundColor: '#2196F3' }}
+                onPress={() => {
+                  setModalManagerVisible(false);
+                  setModalOpenVisible(true);
+                }}
+              >
+                <Text style={styles.textStyle}>{`${I18n.t('animationEditor.drillManager.load')}`}</Text>
+              </TouchableHighlight>
+            ) : (
+              <View />
+            )}
+
+            {/* Delete button */}
+            {props.customeAnimations !== undefined &&
+            props.customeAnimations !== null &&
+            props.customeAnimations.length > 0 ? (
+              <TouchableHighlight
+                style={{ ...styles.openButton, backgroundColor: '#2196F3' }}
+                onPress={() => {
+                  setModalManagerVisible(false);
+                  setModalDeleteVisible(true);
+                }}
+              >
+                <Text style={styles.textStyle}>{`${I18n.t('animationEditor.drillManager.delete')}`}</Text>
+              </TouchableHighlight>
+            ) : (
+              <View />
+            )}
+
+            {/* Cancel button */}
+            <TouchableHighlight
+              style={{ ...styles.openButton, backgroundColor: '#2196F3' }}
+              onPress={() => {
+                setModalManagerVisible(false);
+              }}
+            >
+              <Text style={styles.textStyle}>{`${I18n.t('animationEditor.drillManager.cancel')}`}</Text>
+            </TouchableHighlight>
+          </View>
+        </View>
+      </Modal>
+
+      {/* Modal to delete an existing animations */}
+      <Modal visible={modalDeleteVisible}>
```

Comment:
I haven't read all the new code, but do you think we could  do the same with only one modal, and change it's content ?

Reply:
I guess we could. How do we do it? We have 1 component for each possible content and in the return() method of AnimationEditorPage we use the syntax {isMainModal? <DrillManagerModal/> : isTitleModal? <TitleDrillManagerModal/> : ...
 ?

Reply:
Or we could have a Modal which renders a component, and this component takes care of it all

Something like 
```jsx
      <Modal visible={isModalVisible}>
        <NewDrillManager animation={animation} blahblah={blehbleh}>
      </Modal>
```

So that the AnimationEditorPage stays small and easy to read